index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="node_modules/bootstrap-social/bootstrap-social.css" />
+    <title>Calculator</title>
+</head>
+
+<body>
+    <header style="background-color: goldenrod;" class="jumbotron jumbotron-fluid">
+        <div class="container">
+            <h1>Versatile Carculator</h1>
```

Comment:
Not to be nitpicky but you misspelled "Calculator"

Reply:
HAHA :-) you are right :-) yes its not carculator I do not want to calculate any cars :)

Reply:
I just noticed in my second tab TextToHex I have a lot of typos, which I have to correct.

Reply:
It's okay. We only had two hours to do it. Haha. I am so sure I have errors as well.